server/src/services/databaseOrm.py

```diff
@@ -43,40 +43,40 @@ class Ingest(Base):
     policeprecinct = Column(String)
 
 
-insertFields = {'srnumber': String,
+insertFields = {'srnumber': String(50),
```

Comment:
does this style work with postgres too? Or is this MySQL specific?

Reply:
I don't know about postgres since postgres doesn't require varchar specifications, but this appears to be how you would work with mysql data types with sqlalchemy

Reply:
Hypothetically it should work. We're just adding specificity. I'm pretty sure that Postgres was able to infer these on the fly, but mySQL seems to want it explicit. I can test after the merge if we need to.

Bigger issue is padding. @rgao Did you make sure that the string values are long enough that we won't get too many errors if we see a string that's just a few characters longer than what we've observed? Also did you make sure we aren't getting any truncations on insert (ie were there fields that were larger than the character limit)?

Reply:
At least for the sample of data I was test with, the varchar has to be set for all the values to fit or else there would be an error. I'll make the padding larger just in case.